src/interface/src/app/app-routing.module.ts

```diff
@@ -62,6 +63,12 @@ const routes: Routes = [
         component: SignupComponent,
         canActivate: [createFeatureGuard('login')],
       },
+      {
+        path: 'thankyou',
+        title: 'Account Validation',
```

Comment:
Should the title be `thank you` or something like that?

Reply:
oops, good catch